tests/integration/components/imgix-image-test.js

```diff
@@ -358,7 +358,38 @@ module('Integration | Component | imgix image', function(hooks) {
           hbs`<div style='width:1250px;height:200px;'>{{imgix-image path='/users/1.png?auto=format'}}</div>`
         );
 
-        expectSrcsTo(this.$, (_, uri) => assert.equal(uri.getQueryParamValue('auto'), 'format'));
+        expectSrcsTo(this.$, (_, uri) =>
+          assert.equal(uri.getQueryParamValue('auto'), 'format')
+        );
+      });
+    });
+
+    module('secure url token', function () {
+      test('is included as a query parameter when configured ', async function (assert) {
+        config.APP.imgix.secureURLToken = 'sometoken';
+
+        await render(
+          hbs`<div style='width:1250px;height:200px;'>{{imgix-image path='/users/1.png'}}</div>`
+        );
+
+        expectSrcsTo(this.$, (_, uri) => {
+          const secureParam = uri.getQueryParamValue('s');
```

Comment:
This fails during a test due to a imgix-core-js behaviour https://github.com/imgix/js-core/blob/main/src/index.js#L89 which appears to be related to https://github.com/imgix/js-core/issues/32 ?

I've tested this in a real app and it works fine but can't figure out why this won't work in tests. I tried adding `md5` as a dependency (which is already brought in transitively) but no luck with getting the test to pass 😞 .

The failure looks like
```
not ok 41 Chrome 96.0 - [undefined ms] - Global error: Uncaught TypeError: md5 is not a function at http://localhost:7357/assets/vendor.js, line 71975
 While executing test: Integration | Component | imgix background > application config > secure url token: is included as a query parameter when configured
    ---
        browser log: |
            testContext: [object Object]
            ERROR: Uncaught TypeError: md5 is not a function at http://localhost:7357/assets/vendor.js, line 71975

    ...
not ok 42 Chrome 96.0 - [85 ms] - Integration | Component | imgix background > application config > secure url token: is included as a query parameter when configured
    ---
        actual: >
            null
        stack: >
            http://localhost:7357/assets/vendor.js:71975
        message: >
            Uncaught TypeError: md5 is not a function
        negative: >
            false
        browser log: |
    ...
```

Any help here would be much appreicated.